experiments/snippets/skuidCustom/skuidCustom.js

```diff
@@ -0,0 +1,1344 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-tabs */
+/* eslint-disable indent */
+/* eslint-disable semi */
+/* eslint-disable max-len */
+/* eslint-disable no-continue */
+/* eslint-disable no-shadow */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-extend-native */
+/* eslint-disable no-undef */
+const $ = skuid.$;
+
+// Custom SKUID Functions
+skuid.custom = {};
+
+// Date.formatDate(date)
+// formats a javascript date as YYYY-MM-DD for use with Salesforce
+Date.formatDate = function (date) {
+	const d = new Date(date);
+	let month = `${(d.getMonth() + 1)}`;
+	let day = `${d.getDate()}`;
+	const year = d.getFullYear();
+
+	if (month.length < 2) {
+		month = `0${month}`;
+	}
+	if (day.length < 2) {
+		day = `0${day}`;
+	}
+
+	return [year, month, day].join('-');
+};
+Date.prototype.formatDate = function () { 
+	return Date.formatDate(this);
+};
+
+// Date.getDateFromSFDate(dateStr)
+// Takes a Salesforce formatted date (YYYY-MM-DD) and converts it to a Javascript Date object
+// returns Javascript date object
+Date.getDateFromSFDate = function (dateStr) {

```

Comment:
skuid also provides a method for doing this: `skuid.time.parseSFDate(dateString)`
https://docs.skuid.com/latest/v1/en/skuid/api/skuid_time.html#skuid.time.parseSFDate

Reply:
Removed from skuidCustom